src/nn/ops/conv2d.ts

```diff
@@ -131,6 +131,65 @@ export class Conv2d extends SingleOutputOperation implements FusedOperation {
     return inputs;
   }
 
+  createOutput(): void {
+    let batches, channels, inputHeight, inputWidth, filterHeight, filterWidth;
+    this.tfPadding_ = utils.getPaddings(this.padding_);
```

Comment:
Why do we use tfPadding here? Could we just use padding?

Reply:
Updated in new commit, thanks.